extras/scripts/disable-multipath.sh

```diff
@@ -1,11 +1,46 @@
 #!/usr/bin/env bash
+#
+# stop and disable multipathd (the daemon) *and* blacklist all device names like 
+# /dev/sda so they do not appear in the output of multipath (the CLI)
+#
 
+# shell exit with error (errexit) if any command returns exit != 0 or any variable
+# is undefined (nounset)
+set -ue
+
+warn(){
+echo <<\EOF >&2
```

Comment:
I did mean to use `cat` and I've pushed the correction to [qrkourier/gdeploy/development](https://github.com/qrkourier/gdeploy/commit/291af937ab4b635910a7b12b2f5ab78d10fee04e).